clusterloader2/pkg/measurement/common/wait_for_pods.go

```diff
@@ -91,10 +91,12 @@ func waitForPods(clientSet clientset.Interface, namespace, labelSelector, fieldS
 				runningPodsCount++
 			}
 		}
-		if namespace == metav1.NamespaceAll {
-			glog.Infof("WaitForRunningPods: running %d / %d", runningPodsCount, desiredPodCount)
-		} else {
-			glog.Infof("WaitForRunningPods: %s: running %d / %d", namespace, runningPodsCount, desiredPodCount)
+		if log {
```

Comment:
In general, we're using glog [verbosity](https://godoc.org/github.com/golang/glog#V) to control amount of logs to print - that precisely seems to be the use-case here. So I suggest removing this bool checks and just log those statements at appropriate verbosity (maybe V(4)?).

Reply:
TBH, I wanted to disable log for `waitForPods` if it called by other measurement.
For example this function is called by `waitForControlledPodsRunningMeasurement` in parallel, so it will create many log entries and I'm not sure it those logs will be helpful.
I agree that verbosity can be applied here, but I'm not sure whether I want to remove the switch.
WYT?

Reply:
If you think those logs aren't really helpful, then you can remove it. If you're think they're helpful and there's no measurement wanting to turn it off, then there's no need to add such extra switch atm.

Reply:
The thing is that this log are or are not helpful depending on use case. For example:
- If we use `wait_for_pods` measurement, you have one group of pods you're waiting for. In this case, this logs are very helpful, cause it shows how this specific group becomes ready.
- If we use `wait_for_controlled_pod`, `waiting_for_pod` is just a subfunction. In this case we have many pod groups, so seeing the changes in every group (especially if it's in parallel) is not so helpful. IMHO  it's better to observer only number of controlling object being ready.

Reply:
I'm ok with keeping it this way.